### PR TITLE
Add notebook for image processing using beam

### PR DESCRIPTION
**Pull request by reeba212**
This notebook shows how to use Apache Beam in image processing and can be used as a reference guide for users. It contains an end-to-end workflow that includes
1. Importing and preprocessing the CIFAR-10 dataset
2. Training a model locally using TensorFlow
3. Saving the model to Google Cloud Storage
4. Building a Beam pipeline to preprocess input images and run inference on them

### CODE REVIEW

**Comment by pabloem** [comment]
RAT failures are because files might be missing the Apache License header. See how it's added in other files: https://github.com/apache/beam/blob/a1587e72bdd19f6d47b9b7492cd6b56b83393b80/examples/notebooks/beam-ml/automatic_model_refresh.ipynb

File: examples/notebooks/beam-ml/image_processing_tensorflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,1145 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c5x1HD_V4-RS"
+      },
+      "source": [
+        "## Building a Beam Pipeline\n",
+        "\n",
+        "A Pipeline represents the workflow of a series of computations. Here we are performing the following tasks in our pipeline:\n",
+        "* Creating a PCollection of the data on which we need to run inference\n",
+        "* Appying the Image Preprocessing DoFns we defined earlier \\\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/image_processing_tensorflow.ipynb:935]:
```suggestion
        "* Appying the Image Preprocessing DoFns we defined earlier \n",
```

File: examples/notebooks/beam-ml/image_processing_beam.ipynb [hunk 2]
```diff
@@ -0,0 +1,933 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8gz7_SvN-P2L"
+      },
+      "source": [
+        "## Training a Convolutional Neural Network\n",
+        "\n",
+        "A Convolutional Neural Network (CNN) is one of the most popular model for image processing. Here is a brief description of the convolutional layers used in the model.\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/image_processing_beam.ipynb:468]:
```suggestion
        "A Convolutional Neural Network (CNN) is one of the most popular model types for image processing. Here is a brief description of the convolutional layers used in the model.\n",
```

File: examples/notebooks/beam-ml/image_processing_beam.ipynb [hunk 3]
```diff
@@ -0,0 +1,933 @@
...
+        "* **Reshape**: Changes the shape of the input data to the desired size. <br>\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/image_processing_beam.ipynb:469]: The bullet, line break format here creates weird formatting on GitHub. Could we just get rid of the `<br>` tags?

File: examples/notebooks/beam-ml/image_processing_beam.ipynb [hunk 4]
```diff
@@ -0,0 +1,933 @@
...
+      "metadata": {
+        "id": "NsNImDL8TGM1"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Image Processing using Apache Beam\n",
```
Comment (damccorm) [inline examples/notebooks/beam-ml/image_processing_beam.ipynb:34]: Could you please add colab/github links like our other notebooks, for example:

https://github.com/apache/beam/blob/c2de3391bce1165e0e835b1475cfb254382cdad0/examples/notebooks/beam-ml/automatic_model_refresh.ipynb#L52

image

Note that the links should go to apache/beam master's version of these on colab/github even though those links won't exist until the PR is merged

**Review by damccorm — APPROVED** [review]
Thanks, this looks great!

**Pull request merged**